e2e/cypress/integration/enterprise/cloud/billing/billing_history_free_trial_spec.js

```diff
@@ -0,0 +1,63 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Group: @cloud_only @cloud_trial
+// Skip:  @headless @electron // run on Chrome (headed) only
+
+const dayJs = require('dayjs');
+
+describe('System Console - Billing History section', () => {
+    before(() => {
+        // * Check if server has license for Cloud
+        cy.apiRequireLicenseForFeature('Cloud');
+
+        // # Visit the billing history url
+        cy.visit('admin_console/billing/billing_history');
+
+        // * Check for billing history header
+        cy.get('.admin-console__header').should('be.visible').and('have.text', 'Billing History');
+    });
+
+    it('MM-37053 Finding the default "Cloud Starter" record in billing history screen', () => {
+        const currentDate = new Date();
+
+        // * Check for default record in grid
+        cy.contains('td:nth-of-type(1) span', dayJs(new Date(currentDate.getFullYear(), currentDate.getMonth(), 1)).format('MM/DD/YYYY')).should('exist');
+        cy.contains('td:nth-of-type(2) div', 'Cloud Starter').should('exist');
```

Comment:
Was thinking about this and I thought this should not be specific to "Cloud Starter" only.

You may [get all cloud products](https://api.mattermost.com/#operation/GetCloudProducts) then get specific product's name by comparing the [subscription](https://api.mattermost.com/#operation/GetSubscription). Something like,
```javascript
    let productSubscription;

    before(() => {
        // ...
        cy.apiGetCloudProducts().then(({products}) => {
            cy.apiGetCloudSubscription().then(({subscription}) => {
                productSubscription = products.find((product) => product.id === subscription.product_id);
            });
        });
        // ...
    });
```

Then this can be
```suggestion
        cy.contains('td:nth-of-type(2) div', productSubscription.name).should('exist');
```


Please create new API commands and add to the new `cypress/support/api/cloud.js`. Also, please ensure corresponding typings at `cypress/support/api/cloud.js`
```javascript
Cypress.Commands.add('apiGetCloudProducts', () => {
    return cy.request({
        headers: {'X-Requested-With': 'XMLHttpRequest'},
        url: '/api/v4/cloud/products',
        method: 'GET',
    }).then((response) => {
        expect(response.status).to.equal(200);
        return cy.wrap({products: response.body});
    });
});

Cypress.Commands.add('apiGetCloudSubscription', () => {
    return cy.request({
        headers: {'X-Requested-With': 'XMLHttpRequest'},
        url: '/api/v4/cloud/subscription',
        method: 'GET',
    }).then((response) => {
        expect(response.status).to.equal(200);
        return cy.wrap({subscription: response.body});
    });
});
```